client/lib/products-values/constants.js

```diff
@@ -141,12 +157,15 @@ export const getJetpackProductsDisplayNames = () => {
 		[ PRODUCT_WPCOM_SEARCH_MONTHLY ]: search,
 		[ PRODUCT_JETPACK_SCAN ]: scanDaily,
 		[ PRODUCT_JETPACK_SCAN_MONTHLY ]: scanDaily,
+		[ PRODUCT_JETPACK_ANTI_SPAM ]: antiSpam,
+		[ PRODUCT_JETPACK_ANTI_SPAM_MONTHLY ]: antiSpam,
 	};
 };
 
 export const getJetpackProductsTaglines = () => {
 	const searchTagline = translate( 'Search your site.' );
-	const scanTagline = 'Scan your site.';
+	const scanTagline = translate( 'Scan your site.' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.